packages/grafana-ui/src/components/Logs/LogRowMessage.tsx

```diff
@@ -34,8 +33,8 @@ interface Props extends Themeable {
   updateLimit?: () => void;
 }
 
-const getStyles = stylesFactory((theme: GrafanaTheme) => {
-  const outlineColor = tinycolor(theme.colors.dashboardBg).setAlpha(0.7).toRgbString();
+const getStyles = (theme: GrafanaTheme2) => {
+  const outlineColor = tinycolor(theme.colors.border.weak).setAlpha(0.7).toRgbString();
```

Comment:
> Switching between dark and light mode also gives different colors for highlighting. @ifrost do you know if there are different styles that are being applied for each theme? I can't find anything here. From reading the code it feels like the colours should be the same in both dark and light theme

@olbo98 It's caused by this change. I think it should be:

```suggestion
  const outlineColor = tinycolor(theme.components.dashboard.background).setAlpha(0.7).toRgbString();
```

Reply:
Yeah that seems to be the correct value but it didn't change anything for me.

Reply:
Hmm... I check it locally and it seems to work. `theme.colors.border.weak` has a totally different color, e.g. in dark 

```
theme.colors.border.weak              = rgba(204, 204, 220, 0.07)
theme.components.dashboard.background = rgb((17,18,23)
```